regexes/device/mobiles.yml

```diff
@@ -892,9 +904,12 @@ Casio:
 
 # Cat
 Cat:
-  regex: 'Cat ?(tablet|stargate|nova)'
+  regex: 'Cat ?(tablet|stargate|nova)|B15Q'
   device: 'tablet'
   models:
+    - regex: '(B15Q)'
+      model: '$1'
```

Comment:
Wouldn't use any expressions here, simply:
```
- regex: 'B15Q'
  model: 'B15Q'
```
Won't change anything, but is easier to read (and slightly faster)